corehq/apps/export/tests/test_get_export_file.py

```diff
@@ -676,27 +676,27 @@ class ExportTest(SimpleTestCase):
     @classmethod
     def setUpClass(cls):
         super(ExportTest, cls).setUpClass()
+        with patch('corehq.pillows.utils.get_user_type', return_value='CommCareUser'):
+            case = new_case(
+                case_id='robin',
+                name='batman',
+                case_json={"foo": "apple", "bar": "banana", "date": '2016-4-24'},
+            )
+            case_adapter.index(case.to_json(), refresh=True)
 
-        case = new_case(
-            case_id='robin',
-            name='batman',
-            case_json={"foo": "apple", "bar": "banana", "date": '2016-4-24'},
-        )
-        case_adapter.index(case.to_json(), refresh=True)
-
-        case = new_case(
-            owner_id="some_other_owner",
-            case_json={"foo": "apple", "bar": "banana", "date": '2016-4-04'},
-        )
-        case_adapter.index(case.to_json(), refresh=True)
+            case = new_case(
+                owner_id="some_other_owner",
+                case_json={"foo": "apple", "bar": "banana", "date": '2016-4-04'},
+            )
+            case_adapter.index(case.to_json(), refresh=True)
```

Comment:
```suggestion
            case_adapter.index(case, refresh=True)
```

Reply:
nit: you could use `case_adapter.bulk_index`

Reply:
https://github.com/dimagi/commcare-hq/pull/32660/commits/db1087b79ae0d50322410862443f03e2a20eeb21